static/templates/settings/alert_word_settings.hbs

```diff
@@ -27,5 +27,14 @@
             </div>
         </div>
     </form>
-    <ul id="alert_words_list"></ul>
+    <div class="progressive-table-wrapper" data-simplebar>
+        <table class="table table-condensed table-striped wrapped-table">
+            <thead>
+                <th class="active alert-word" data-sort="alphabetic" data-sort-prop="full_name">{{t "Word" }}</th>
+                <th class="user_role alert-word-delete" data-sort="role">{{t "Delete" }}</th>
+            </thead>
+            <tbody id="alert_words_list" class="alert_words_list required-text thick"
+              data-empty="{{t 'No users match your current filter.' }}"></tbody>
```

Comment:
Alert words are not users and cannot be filtered.

Reply:
Any suggestions on a replacement message "Currently no alert words"?